### PR TITLE
Feat/remove time server

### LINKED ISSUE #66: Move time server designs to deployment considerations...
As was resolved in the Uptane workshop today, we will move the time server mechanisms to the deployment considerations.  The spec will instead state why a notion of wall clock time is important for checking expiry of certificates and point to the deployment considerations for options.

@awwad, Would you kindly take a first crack at this?  Please remember to leave a place for Ira to add text about the secure time options he is aware of.

### PR DESCRIPTION
**Pull request by tkfu**
Closes #66

### CODE REVIEW

**Comment by jhdalek55** [comment]
Can someone please review this PR so we can close out Issue #66?

File: uptane-standard.md [hunk 1]
```diff
@@ -738,24 +716,13 @@ An ECU version report is a metadata structure that MUST contain the following in
   * The signature of the hash
 * A payload containing:
   * The ECU's unique identifier (e.g. the serial number)
-  * The latest time downloaded from the time server, if the time server is implemented
```
Comment (JustinCappos) [inline uptane-standard.md]: Is some notion of time needed / useful here?  It's a matter of saying when this data was generated.  I suppose you are saying that the nonce covers this, but perhaps being clearer about the choices.

**Comment by JustinCappos** [comment]
@tkfu I think my comment is really quick to fix.  Let me know when you're done and I can approve and merge the PR.

**Comment by JustinCappos** [comment]
@tkfu: gentle nudge.  I'm hoping we can resolve this soon

**Comment by tkfu** [comment]
> @tkfu: gentle nudge. I'm hoping we can resolve this soon

Yup, sorry about that. Does 4391470 resolve it?

**Comment by JustinCappos** [comment]
Looks good!  Thanks!

**Pull request merged**